packages/fx-core/src/plugins/resource/function/plugin.ts

```diff
@@ -226,6 +230,13 @@ export class FunctionPluginImpl {
     });
 
     if (func.method === "addResource") {
+      const functionNameQuestion: TextInputQuestion = {
+        name: QuestionKey.functionName,
+        title: InfoMessages.askFunctionName,
+        type: "text",
+        default: ctx.projectSettings?.defaultFunctionName ?? DefaultValues.functionName,
```

Comment:
I didn't get the purpose of this change. If we set `default` as `projectSettings.defaultFunctionName`, toolkit always throws error telling Function already exists.
![image](https://user-images.githubusercontent.com/26134943/129654510-71caa1a1-513b-4861-8c09-3b32954b0b6e.png)

Reply:
Toolkit will also throw error if use constant value "getUserProfile" when user already has a function named `getUSerProfile`.
I think both the constant value "getUserProfile" as well as the config value for `defaultFunctionName` doesn't make much sense in the get question name on add resource. Let's me change back to the original logic.